.github/workflows/run-minor-release.yml

```diff
@@ -5,7 +5,7 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: 'The version (semver format: major.minor.patch)'
+        description: 'The version (semver format: major.minor.0)'
```

Comment:
to help with the review:
minor release are always `major`.`minor`.`0`, hence the patch is not required.